### DISCUSSION TITLE
Unable to Create a new Codespace in VS code

### DISCUSSION THREAD

**Discussion by Jethro-magaji** [community/community#3904] [Codespaces]
I could not create a new Codespace in VS code, it gives me this error;
"Create New Codespace: List instances request failed with status 404. Correlation ID: C85B:A029:7E7BE6:80F0B8:60947DC4"

**Comment by 2percentsilk** [comment]
Hi there,

If you go to www.github.com/codespaces when you're signed in, are you able to create a codespace from there?

**Comment by reeceatkinson** [comment]
Are you able to start a codespace from the repo?

**Reply by rmn30** [reply]
I go this same error after pressing '.' in a github repo with a dev container then choosing 'Create new dev container'. I am able to create one from github.com/codespaces but it used to work until recently from the vs code in browser...
